source/imports/domain/banking-domain-module.js

```diff
@@ -0,0 +1,15 @@
+import BankAccountRouter from '../es-routers/bank-account-router';
+
+const BankingDomain = Space.Module.define('BankingDomain', {
+
+  requiredModules: [
+    'Space.eventSourcing'
+  ],
+
+  singletons: [
+    BankAccountRouter
```

Comment:
Routers are an Application concern, so we have a little friction with the current structure. I'd suggest we even just omit the use of a Space.Module for this project, as we just have a single bounded context. However, for the purpose of a larger application I would invert the directories to read:
`imports/context/domain/my-aggregate`
`imports/context/application/command-handlers/…`
`imports/context/infrastructure/…`
`imports/context/module.js`